__tmp_ws_base.py

```
@@ -0,0 +1,589 @@
+                websockets.exceptions.WebSocketException,
+                json.JSONDecodeError,
+                WSClientException,
+            ) as e:
+                logger.error("Exception in message handler: %s", e)
+                self._background_exception = WSClientException(
+                    f"Exception in message handler: {e}"
+                )
+                break
+
+    def _build_subscription_message(
+        self, product_ids: List[str], channel: str, message_type: str, public: bool
```

Ruff (0.14.3)

Unused method argument: `public`

(ARG002)